test/test_NumericRounder.py

```diff
@@ -16,10 +16,13 @@
     (Decimal('-1.12345'), Decimal('-1.123')),
     (None, None),
     ('str', 'str'),
+    (True, True),
+    (False, False)
```

Comment:
Makes sense to me as long as you meant comma `,`, otherwise I don't understand :)